source/mainnet/net/concepts/concepts-baker.rst

```diff
@@ -76,3 +85,111 @@ Finalization committee
+
+   You'll need to install and set up the Desktop Wallet to create and manage identities and accounts and add a baker. See :ref:`Set up the Desktop Wallet<overview-desktop>`.
+
+.. dropdown:: Step 4: Set up an identity and an initial account
+
+   Once you've installed the Desktop Wallet, you must set up an identity and an initial account. You may want to create a separate account to use as a baker account, since the Identity Provider knows the user who submits the initial account to the chain. See :ref:`Create an identity and an initial account in the Desktop Wallet <create-initial-account>` and :ref:`Create an account in the Desktop Wallet<create-account>`.
+
+.. dropdown:: Step 5: Add a baker in the Desktop Wallet
+
+   You're now ready to add a baker in the Desktop Wallet and generate baker keys. This process varies depending on whether you need one or more signatures before you can submit the transaction to the chain. See :ref:`Add a baker account in the Desktop Wallet <add-baker-mw>`. You can also change the number of signatures required on an account before a transaction can be submitted to the blockchain. See :ref:`Change the signature threshold <guide-change-signature>`
+
+.. dropdown:: Step 6: Configure the node with the baker keys
+
+   The last step is to configure the running node with the baker keys so the node can start baking. You can also choose to have a third-party node runner run a node for you if you do not want to run the node yourself; in this case you will need to provide your baker keys to the node runner in a secure manner.
+
+   - :ref:`On Windows<baker-windows>`
+
+   - :ref:`On macOS<baker-macos>`
+
+   - :ref:`On Ubuntu<baker-Ubuntu>`
+
+   - :ref:`On Docker/Linux<baking-docker>`.
+
+For information about how to update your baker or stop baking, see :ref:`Change baker options<update-baker-mw>`.
+
+Baking with |mw-gen1| and |mw-gen2|
+===================================
+
+This overview describes the recommended scenario for running a node and becoming a baker on the Concordium blockchain when using |mw-gen1| or |mw-gen2| and running a node.
+
+.. dropdown:: Step 1: Set up the node
+
+   For baking you must be running a node on the Concordium blockchain. You can run a node :ref:`on Windows<run-node-windows>`, :ref:`on macOS<run-node-macos>`, :ref:`on Ubuntu<run-node-ubuntu>` or using :ref:`Docker<run-a-node>`. You can also have a third-party run a node on your behalf.
+
+.. dropdown:: Step 2: Set up the Wallet
+
+   The |mw-gen1| is available for iOS and Android. |mw-gen2| is available for Android. For instructions about download and setup, see :ref:`setup-mobile-wallet`.
+
+.. dropdown:: Step 3: Set up an identity and initial account
+
+   Once you've installed the Wallet, you must set up an identity and an account. If using |mw-gen1| it is recommended to create a separate account to use as a baker account. For instructions, see :ref:`create-initial-account` and :ref:`create-account`.
```

Comment:
It is because I repurposed the old topic name instead of renaming the file and having to correct a lot of cross references. The reader does not see it unless they look at the path in the URL.